### DISCUSSION TITLE
Customer Certificate Extension, how do in input the Value, with with which Encoding

### DISCUSSION THREAD

**Discussion by 8805102** [Keyfactor/ejbca-ce#280] [Q&A] [answered]
I'm trying to input a value for Subject Information Access (1.3.6.1.5.5.7.1.11).
In an OpenSSL configuration file it looks like this (as on way).

[ xca_extensions ]
keyUsage=critical,keyCertSign, cRLSign
subjectKeyIdentifier=hash
basicConstraints=critical,CA:TRUE
subjectInfoAccess = @sia_section

[ sia_section ]
1.3.6.1.5.5.7.48.5;URI  = http://example.org/rpki/root/
1.3.6.1.5.5.7.48.1;URI  = http://example.org/

How does this get input into the UI for custom certificate extensions and with what encoding?

Thank you,
Mark

**Comment by primetomas** [comment, marked as answer]
Look here: https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/certificate-profiles-overview/custom-certificate-extensions#CustomCertificateExtensions-CreatingaCustomExtensionfromExistingCertificate

Cheers,
Tomas

**Reply by 8805102** [reply]
Perfect.  Exactly what I needed, I kept try various combos of values and encoding, and using DEROBJECT with the hex value did it.
Thank you
